graphics/src/AssimpLoader.cc

```diff
@@ -695,12 +706,18 @@ Mesh *AssimpLoader::Load(const std::string &_filename)
   }
   auto& rootNode = scene->mRootNode;
   auto rootName = ToString(rootNode->mName);
-
-  // compute assimp root node transform
-  std::string extension = _filename.substr(_filename.rfind(".") + 1,
-      _filename.size());
+  auto fileBaseName = common::basename(_filename);
+  std::string extension;
+  std::size_t extIdx = _filename.rfind(".");
```

Comment:
include `<cstddef>` and `<string>`

Reply:
added. [163222b](https://github.com/gazebosim/gz-common/pull/606/commits/163222b1bf446210069db05f8742220981d2b4ef)